wiki/tools/mapviz.md

```diff
@@ -0,0 +1,352 @@
+# Mapviz for Map Based Visualization in ROS2 
```

Comment:
Missing page meta information. Please see the template.

Also, the first paragraph shouldn't have a header (i.e. "# Mapviz...") or any markdown. Jekyll uses the first paragraph (or the first 100 words) to generate a blurb for your entry. Unfortunately, it pulls what it finds verbatim.

CC: @nevalsar